src/ngraph/ngraph_emitter.cc

```diff
@@ -686,6 +686,25 @@ void Emitter::CreateLayerOps() {
                               squeeze_axis && (slice_step == 1));
   };
 
+  // stack takes a list of tensors of equal shape and
+  // concatenates them along a given axis expanded for each input
+  ngraph_op_funcs_["stack"] = [this](const NodePtr& node) {
+    // get the concat axis
+    size_t axis = get_default_transformed_axis(node, "axis", 0,
+                                               node->inputs_[0]->shape_.ndim() + 1);
+    auto shape = op_map_[node->inputs_[0]]->get_shape();
+    shape.insert(shape.begin() + axis, 1);
+    // grab in input ngraph nodes and Reshape them
```

Comment:
NBD, but IMO the code below is quite readable and so this comment (line 697) isn't necessary.

Also, is "in input" a typo?

Reply:
Yep